starter-code/src/viking.js

```diff
@@ -1,11 +1,94 @@
-// Soldier
-function Soldier() {}
+// SOLDIER
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+}
+Soldier.prototype.attack = function(){
+    return this.strength;
+};
+Soldier.prototype.receiveDamage = function(damage){
+    this.health -= damage; 
+};
 
-// Viking
-function Viking() {}
+Soldier.prototype.battleCry = function() {
+    return "Odin Owns You All!"
+}
+
+//VIKING
+function Viking(name, health, strength) {
+    Soldier.call(this, health, strength);
+    this.name = name;
+}
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+
+Viking.prototype.receiveDamage = function(damage) {
+    Soldier.prototype.receiveDamage.call(this, damage);
+    
+    if (this.health > 0) {
+        return this.name + " has received " + damage + " points of damage"        
+    } else {
+       return  this.name + " has died in act of combat"
+    }
+}
```

Comment:
Fijáos que esta función ya está heredada de `Soldier`. Si lo haceis así, estáis machacando la función heredada. Necesitais modificar la función heredada, y se hace de la siguiente manera -> `Soldier.prototype.receiveDamage.call(this, damage)` dentro de `Viking.prototype.receiveDamage(damage)`